dev-libs/libmelf/libmelf-0.4.0-r1.ebuild

```diff
@@ -11,30 +11,32 @@ SRC_URI="http://www.hick.org/code/skape/${PN}/${P}.tar.gz"
 LICENSE="Artistic"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
+IUSE="static-libs"
 
 DEPEND=""
 RDEPEND="${DEPEND}"
 
-src_prepare() {
-	# This patch was gained from the elfsign-0.2.2 release
-	epatch "${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
-	# Cleanup stuff
-	epatch "${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
+	"${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
+)
+
+src_configure() {
+	default
```

Comment:
default not needed (no `econf` available)

Reply:
If I remove it I am getting errors like

make -j8 CC=x86_64-pc-linux-gnu-gcc 'OPTFLAGS=-march=native -O2 -pipe -fPIC' 
make: *** No targets specified and no makefile found.  Stop.
 * ERROR: dev-libs/libmelf-0.4.0-r1::gentoo failed (compile phase):
 *   emake failed